pychromecast/socket_client.py

```diff
@@ -212,6 +217,21 @@ def __init__(self, host, port=None, cast_type=CAST_TYPE_CHROMECAST,
                                  NetworkAddress(self.host, self.port)))
             raise
 
+    def _get_host_from_service(self, service):
```

Comment:
This wrapper feels weird. It adds a code indirection that is a bit confusing. Can't we have the caller of this method directly call `get_host_from_service` ? 

I also think that in the `__init__`, we should either accept a host/port or a `service`.

Reply:
The wrapper is dumb, but without it pylint complains `initialize_connection()` was too long. OK to silence that?

Ok, will fix `__init__` as you suggest.

Reply:
yeah silence the pylint error

Reply:
OK, fixed.